apps-rendering/src/bodyElement.ts

```diff
@@ -124,6 +126,14 @@ interface NewsletterSignUp extends Omit<Newsletter, 'theme'> {
 	theme: ArticleTheme;
 }
 
+type Callout = {
```

Comment:
How has this model been arrived at? I think it might be missing / gaining a few fields

Reply:
The nested `campaign` field has been removed and replaced it with some fields specific to callouts. My thinking is that our internal article model (`Item`) and the rendering code probably doesn't need to care about the existence or structure of campaigns? It just needs some information about callouts so it can render them correctly. Also, the `Campaign` type now reflects the full set of possible campaigns - rather than just callout campaigns specifically - so it may not make sense to include it as a field on a `Callout` model?

Regarding missing fields, do you have an example? I haven't included the changes you've made in #6333 (e.g. the collapsible information), because I didn't want to couple this PR to that work (it's already doing quite a lot!). I'm anticipating that those changes can be made separately once the CAPI work has been completed?

Reply:
Thanks for this Jamie! this makes sense, it's nice to have them separated. 

In terms of missing fields, I think I might be conflating this work and the work that I did which, as you say, needs refactoring on top of this work and can be done separately.